exporters/dd.py

```diff
@@ -0,0 +1,111 @@
+import logging
+from enum import Enum
+from threading import Thread
+from time import sleep
+from typing import Union
+
+from celery.events.state import Task, Worker
+from celery.states import FAILURE, PENDING, READY_STATES, STARTED, SUCCESS
+from datadog import initialize, statsd
+
+from exporters import Exporter
+
+logger = logging.getLogger(__name__)
+
+
+class DataDogMetrics(Enum):
+    TASK_WAIT_TIME = "celery.task.wait_time"
+    TASK_RUNTIME_TIME = "celery.task.run_time"
+    TOTAL_SUCCESS = "celery.task.success_count"
+    TOTAL_FAILED = "celery.task.fail_count"
+
+
+class DataDogSummary:  # maybe this can be generic
+    def __init__(self, events):
+        self.events = events
+
+    @property
+    def wait_time(self):
+        """
+        Returns wait time in seconds
+
+        :return: int
+        """
+        try:
+            client_sent_time = self.events[PENDING]["timestamp"]
```

Comment:
should this be `received` events

observation task started from celery-beat don't have `pending` or `task-sent` event

Reply:
^consider adding this as a comment on the code, not just the PR